src/Faker/Provider/en_PH/Address.php

```diff
@@ -392,9 +392,153 @@ class Address extends \Faker\Provider\en_US\Address
+        'General Luna, Upper',
+        'General Luna, Lower',
+        'Gibraltar',
+        'Greenwater Village',
+        'Guisad Central',
+        'Guisad Sorong',
+        'Happy Hollow',
+        'Happy Homes - Happy Homes-Lucban',
+        'Harrison-Claudio Carantes',
+        'Hillside',
+        'Holy Ghost Extension',
+        'Holy Ghost Proper',
+        'Honeymoon - Honeymoon-Holy Ghost',
+        'Imelda R. Marcos - La Salle',
+        'Imelda Village',
+        'Irisan',
+        'Kabayanihan',
+        'Kagitingan',
+        'Kayang Extension',
+        'Kayang-Hilltop',
+        'Kias',
+        'Legarda-Burnham-Kisad',
+        'Liwanag-Loakan',
+        'Loakan Proper',
+        'Lopez Jaena',
+        'Lourdes Subdivision Extension',
+        'Lourdes Subdivision, Lower',
+        'Lourdes Subdivision, Proper',
+        'Lualhati',
+        'Lucnab',
+        'Magsaysay Private Road',
+        'Magsaysay, Lower',
+        'Magsaysay, Upper',
+        'Malcolm Square-Perfecto - Jose Abad Santos',
+        'Manuel A. Roxas',
+        'Market Subdivision, Upper',
+        'Middle Quezon Hill Subdivision - Quezon Hill Middle',
+        'Military Cut-off',
+        'Mines View Park',
+        'Modern Site, East',
+        'Modern Site, West',
+        'MRR-Queen of Peace',
+        'New Lucban',
+        'Outlook Drive',
+        'Pacdal',
+        'Padre Burgos',
+        'Padre Zamora',
+        'Palma-Urbano - Cariño-Palma',
+        'Phil-Am',
+        'Pinget',
+        'Pinsao Pilot Project',
+        'Pinsao Proper',
+        'Poliwes',
+        'Pucsusan',
+        'Quezon Hill Proper',
+        'Quezon Hill, Upper',
+        'Quirino Hill, East',
+        'Quirino Hill, Lower',
+        'Quirino Hill, Middle',
+        'Quirino Hill, West',
+        'Quirino-Magsaysay, Upper - Upper QM',
+        'Rizal Monument Area',
+        'Rock Quarry, Lower',
+        'Rock Quarry, Middle',
+        'Rock Quarry, Upper',
+        'Saint Joseph Village',
+        'Salud Mitra',
+        'San Antonio Village',
+        'San Luis Village',
+        'San Roque Village',
+        'San Vicente',
+        'Sanitary Camp, North',
+        'Sanitary Camp, South',
+        'Santa Escolastica',
+        'Santo Rosario',
+        'Santo Tomas Proper',
+        'Santo Tomas School Area',
+        'Scout Barrio',
+        'Session Road Area',
+        'Slaughter House Area - Santo Niño Slaughter',
+        'SLU-SVP Housing Village',
+        'South Drive',
+        'Teodora Alonzo',
+        'Trancoville',
+        'Victoria Village'
+        //End of Baguio City Barangay
+
+    );
+    
+    protected static $purok = array(
+        'Purok 1', 'Purok 2', 'Purok 3', 'Purok 4', 'Purok 5',
+        'Purok 6', 'Purok 7', 'Purok 8', 'Purok 9', 'Purok 10',
+        'Purok 11', 'Purok 12', 'Purok 13', 'Purok 14', 'Purok 15',
+        'Purok 16', 'Purok 17', 'Purok 18', 'Purok 19', 'Purok 20'
+    );
+    
+    public function purok()
```

Comment:
If you ad a new formatter, please add phpDoc explaining what it refers to (preferably with a Wikipedia link), and a note in the Locale-Specific section of the README